src/api.js

```diff
@@ -0,0 +1,88 @@
+const apiPath = '/hr-server/api/v1/';
+
+export const getAllEmployees = async () => {
+    try {
+        const response = await fetch(apiPath + 'employees', {
+            method: 'GET',
+            headers: {
+                Accept: 'application/json'
+            }
+        })
+        return await response.json();
+    } catch (err) {
+        return err;
+    }
+}
+
+export const postAllEmployees = async (employees) => {
+    try {
+        const response = await fetch(apiPath + 'employees', {
+            method: 'POST',
+            headers: {
+                Accept: 'application/json'
+            },
```

Comment:
Okay so this ones my fault. I forgot to mention for the POST requests the headers need an extra part so that the server knows what type of data it is looking at. So have to modify the headers to look like this
```js
headers: {
      Accept: 'application/json', 
     'Content-Type': 'application/json'
}
```